content/kontakt.md

```diff
@@ -8,6 +8,65 @@ weight = 2
 adres: al. Grunwaldzka 4 / 3, 80-236 Gdańsk, Poland       
 współrzędne: 54°22'27.804" N, 18°37'9.829" E      
 
-tel: +48 507 313 631        
+tel: +48 507 313 631
 mail: kontakt@hs3.pl        
 irc: freenode.net/#hs3      
+```
```

Comment:
It does not recive a command prompt styling -> either it shall be made into a pragraph or additonal styling shall be added to custom.css